src/terrama2/services/analysis/core/occurrence/zonal/Operator.hpp

```diff
@@ -48,32 +50,54 @@ namespace terrama2
       {
         namespace occurrence
         {
-          namespace aggregation
+          namespace zonal
           {
+            /*!
+            \brief Calculates the all statistics of the attribute of occurrences in the monitored object area.
+
+            \param statisticOperation The statistic operation called by the script.
+            \param dataSeriesName DataSeries name.
+            \param buffer Buffer to be used in the monitored object.
+            \param dateFilterBegin Begin time filter for the data.
+            \param dateFilterEnd End time filter for the data.
+            \param attribute Name of the attribute to be used in statistic operator.
+            \param aggregationBuffer Buffer configuration to be used to aggregate occurrences in the same area.
+            \param restriction SQL restriction.
+            \return The result of the selected operation.
+          */
+            double operatorImpl(terrama2::services::analysis::core::StatisticOperation statisticOperation,
+                                const std::string &dataSeriesName, terrama2::services::analysis::core::Buffer buffer,
+                                const std::string &dateFilterBegin, const std::string &dateFilterEnd,
+                                terrama2::services::analysis::core::Buffer aggregationBuffer, const std::string &attribute,
+                                terrama2::services::analysis::core::StatisticOperation aggregationStatisticOperation,
+                                const std::string &restriction);
+
+            /*!
+              \brief Calculates the count of occurrences in the monitored object.
+
+              \param dataSeriesName DataSeries name.
+              \param buffer Buffer to be used in the monitored object.
+              \param dateFilter Time filter for the data.
+              \param restriction SQL restriction.
 
-            int count(const std::string& dataSeriesName,
-                      const std::string& dateFilter,
-                      terrama2::services::analysis::core::Buffer aggregationBuffer,
```

Comment:
GitHub is comparing wrong files, zonal/Operator.hpp and aggregation/Operator.hpp are different files